events/eventconsumer/event_consumer.go

```diff
@@ -120,9 +109,6 @@ func (e *EventConsumer) Listen() {
 	// Consume pending events
 	e.ConsumeEventsUntilEmpty(ctx)
 
-	pgNotify := make(chan string)
```

Comment:
The go routines need to be created here - ConsumeEventsUntilEmpty should be sync and not spawn any go routines